misc.txt

```diff
@@ -0,0 +1,83 @@
+List(
+    Label("start"){
+        List(
+            Face(WEST),
+            Go until FRONT_WALL,
+            Face(NORTH),
+            Go until FRONT_WALL,
```

Comment:
I'm guessing it would be something like Go until OPEN_RIGHT_WALL and perhaps having labels for each direction the picobot could be moving to know which way to change the facing to. I wonder how the complexity of writing this compares to the original syntax.